devices/pine64-pinephone-braveheart/kernel/default.nix

```diff
@@ -6,21 +6,21 @@
 }:
 
 (mobile-nixos.kernel-builder {
-  version = "5.5.0";
+  version = "5.7.0";
   configfile = ./config.aarch64;
   src = fetchFromGitLab {
     owner = "pine64-org";
     repo = "linux";
-    rev = "94cf851f0f4443c771a926102dee497def319b49";
-    sha256 = "1a4ch2j8hla3xd7rv38ra6bnv14lsnj0srhlh1c8vxxvwywzg815";
+    rev = "c906a7d36abaa9ca379015c275b85af5d5b01987";
+    sha256 = "0z4j6zd4w1miw5vsfwzxbcrqskav710vxkxbyjrbwmycpzc0jpkb";
   };
   patches = [
-    ./0001-dts-pinephone-Setup-default-on-and-panic-LEDs.patch
+    # ./0001-dts-pinephone-Setup-default-on-and-panic-LEDs.patch
```

Comment:
Obviously, this'll need to be updated.

See #185 which has an appropriate replacement patch, at least for 5.6. I guess it should apply on 5.7 too.